sos/plugins/fwupd.py

```diff
@@ -0,0 +1,45 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+
+
+class Fwupd(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
+    """fwupd information
+    """
+
+    plugin_name = 'fwupd'
+    profiles = ('system', )
+
+    packages = ('fwupd',)
+
+    def setup(self):
+        self.add_cmd_output([
+            "fwupdmgr get-approved-firmware",
+            "fwupdmgr get-devices --no-unreported-check",
+            "fwupdmgr get-history",
+            "fwupdmgr get-remotes",
+            "fwupdmgr get-topology --no-unreported-check"
+        ])
+
+        self.add_copy_spec([
+            "/etc/fwupd",
+            "/usr/share/fwupd",
+        ])
+
+        self.add_journal(units="fwupd")
+
+    def postproc(self):
+        self.do_file_sub(
+            "/etc/fwupd/redfish.conf",
```

Comment:
Nice catch. I think `/etc/fwupd/remotes.d/lvfs-testing.conf` needs the same (maybe some more or even all configs? those two have "Password" present by default).